pythoncz/models/events.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+
+import yaml
+import requests
+import ics
+import arrow
+
+
+feeds_datafile = Path(__file__).parents[1] / "static/data/events_feed.yml"
```

Comment:
```
feeds_datafile = Path(app.root_path) / 'static/data/events_feed.yml'
```

Reply:
maybe the file could be called `events_feeds.yml` as there is more of the feeds than just one?